src/config.h

```diff
@@ -233,24 +243,26 @@ struct Config {
 
   } model;
 
+  // clang-format off
```

Comment:
Avoid setting this formatting off is possible. Could you werite the comments so they look cleaner. Maybe move them to the top of the member name instead of it being after the member name